doc/source/examples/multiphysics/capillary-wave/capillary-wave.rst

```diff
@@ -0,0 +1,379 @@
+================================
+Capillary Wave
+================================
+
+This example simulates the damping of a small amplitude capillary wave for different time-steps allowing to study the capillary time-step constraint. The problem is inspired by the test case of Denner *et al.* `[1] <https://doi.org/10.1016/j.jcp.2022.111128>`_
+
+
+--------
+Features
+--------
+
+- Solver: ``lethe-fluid`` 
+- Volume of fluid (VOF)
+- Unsteady problem handled by an adaptive BDF2 time-stepping scheme
+- Usage of Bash scripts to write, launch and postprocess multiple cases
+- Usage of Python scripts for postprocessing data
```

Comment:
Usage of Bash or "Use of Bash"

Reply:
I think both work, and it's more a question of preferences in this case. "Usage" is the term used in most of our examples. However, I do think "use" is more straightforward and I like it better. We could simply change it through the other examples also. If agreed on, I'll take care of it when I'll continue with the uniformization of the documentation (probably during the upcoming hackathon).

Reply:
I think that's better! I'll change it for that!